README.md

```diff
@@ -2,11 +2,36 @@
 
 A Vob is a "vector of bits": a sequence of bits which exposes a `Vec`-like
 interface. Whereas `Vec<bool>` requires 1 byte of storage per bit, `Vob`
-requires only 1 bit of storage per bit. For larger numbers of bits, Vobs can
-lead to a significant memory decrease and performance increase. `Vob` is broadly
+requires only 1 bit of storage per bit. `Vob` is broadly
 similar to [BitVec](https://crates.io/crates/bitvec-rs), but has an API more
 closely aligned to `Vec<bool>` and allows non-32-bit backing storage. On 64-bit
-systems this can lead to a substantial performance increase.
+systems this can lead to a substantial performance increase, particularly
+when functions such as
+[`iter_set_bits`](https://docs.rs/vob/0.1.0/vob/struct.Vob.html#method.iter_set_bits)
+are used.
 
-`Vob` currently doesn't implement all of the same functions one would find in
-`Vec<bool>`. We welcome contributions!
+
+## Migrating from `Vec<bool>`
+
+As far as possible, `Vob` is intended to have a superset of `Vec<bool>`'s interface, which
```

Comment:
Is "superset" correct here? I thought you had implemented only part of what `Vec` offers. In fact you say that in the next sentence.

Reply:
The weasel words are "intended to" ;)

Reply:
OK, as long as you think it's clear.